### ISSUE TITLE
[bug] [project-references] "tsc --build" fails to report non-existent module errors

### ISSUE THREAD

**Issue by trusktr** [microsoft/TypeScript#38494]
**TypeScript Version:**  3.8.3

**Search Terms:**

"typescript build project references no error on non-existent module"
"typescript build project references no error on incorrect import"
"typescript build project references no error when module is not found"

**Code**

```ts
import "./file-that-does-not-exist" // No error!
```

**Expected behavior:**

There should be an error.

**Actual behavior:**

The next screenshot in VS Code show that if I try to go to the definition of the module, it can't find it (because it doesn't exist, and Webpack also says the same), yet the TypeScript build (`tsc --build`) does not report any error.

![no-error](https://user-images.githubusercontent.com/297678/81641869-e3104480-93d6-11ea-8fee-d13fe19f611e.png)

Is this related to imports that don't create any identifiers?

**Playground Link:** 

I don't have a reproduction, only a private repo at the moment.

**Related Issues:** 

Possibly related: https://github.com/microsoft/TypeScript/issues/38495

**Comment by trusktr** [comment]
I'll just close this and re-open if/when I have a simple reproduction.
